src/App.js

```diff
@@ -1,73 +1,34 @@
-import { loadFromLocal, saveToLocal } from './utilityFunctions/localStorage';
-import { useEffect, useState } from 'react';
-
 import Form from './components/Form';
 import MainWrapper from './components/MainWrapper';
 import RecipeList from './components/RecipeList';
+import { useState } from 'react';
+
+// const fetcher = (...args) => fetch(...args).then(res => res.json());
```

Comment:
```suggestion

```
I guess you can remove that, but maybe you left in on purpose. there are two more a few lines down